tests/acceptance/init/default-test.js

```diff
@@ -0,0 +1,129 @@
+import { pathExists, readFile } from 'fs-extra';
+import { expect, bigtest } from '../helpers';
+import { describe, it, afterEach, beforeEach } from 'mocha';
+import cleanupFiles from '../../../lib/init/utils/clean-up-files';
+
+const CWD = process.cwd();
+
+describe('Acceptance: `bigtest init`', () => {
+  describe('bigtest init', () => {
+    let result;
+
+    beforeEach(async () => {
+      result = await bigtest('init');
+    });
+
+    afterEach(async () => {
+      await cleanupFiles('bigtest', CWD);
+    });
+
+    it('outputs the right message to the console', () => {
+      expect(result.stdout.toString()).to.equal(
+        '\nBigTest has been initialized with @bigtest/react \n\n'
+      );
+    });
+
+    it('creates a bigtest folder', async () => {
+      let hasBigtestFolder = await pathExists(`${CWD}/bigtest`);
+
+      expect(hasBigtestFolder).to.equal(true);
+    });
+
+    it('does not create a bigtest network folder', async () => {
+      let hasNetworkFolder = await pathExists(`${CWD}/bigtest/network`);
+
+      expect(hasNetworkFolder).to.equal(false);
+    });
+
+    it('inits with @bigtest/react', async () => {
+      let helperFile = await readFile(
+        `${CWD}/bigtest/helpers/setup-app.js`,
+        'utf-8'
+      );
+
+      expect(
+        helperFile.includes(
+          "import { setupAppForTesting } from '@bigtest/react';"
+        )
+      ).to.equal(true);
```

Comment:
```
expect(helperFile).to.include("import { setupAppForTesting } from '@bigtest/react';")
```

Otherwise, if this fails we just get `expected false to be true` vs `expected x to include y`